src/uniprotkb/components/entry/SimilarProteins.tsx

```diff
@@ -0,0 +1,123 @@
+import { Loader, Message, Tabs, Tab, Card, Button } from 'franklin-sites';
+import { FC, useMemo } from 'react';
+import { groupBy } from 'lodash-es';
+import { generatePath, Link } from 'react-router-dom';
+import { getClustersForProteins } from '../../../shared/config/apiUrls';
+import useDataApi from '../../../shared/hooks/useDataApi';
+import { UniRefLiteAPIModel } from '../../../uniref/adapters/uniRefConverter';
+import EntrySection, {
+  getEntrySectionNameAndId,
+} from '../../types/entrySection';
+import { Location, LocationToPath } from '../../../app/config/urls';
+
+const SimilarProteins: FC<{
+  isoforms: { isoforms: string[] };
+  primaryAccession: string;
+}> = ({ isoforms, primaryAccession }) => {
+  const searchUrl = getClustersForProteins([
+    primaryAccession,
+    ...isoforms.isoforms,
+  ]);
+  // Get the clusters in which the canonical and isoforms are found
+  const { loading, data, error } = useDataApi<{
+    results: UniRefLiteAPIModel[];
+  }>(searchUrl);
+  const clusterData = useMemo(() => {
+    if (data) {
+      const { results } = data;
+      // Remove all items with only 1 member as it will be canonical/isoform
+      const filtered = results.filter((item) => item.members.length > 1);
+
+      const clusterTypeGroups = groupBy(filtered, 'entryType');
+      // Note: thought we could use Dictionary<T> here but lodash doesn't
+      // seem to export it
+      const allClusterTypesGroups: {
+        [key: string]: { [key: string]: UniRefLiteAPIModel[] };
+      } = {};
+      Object.keys(clusterTypeGroups).forEach((key) => {
+        allClusterTypesGroups[key] = groupBy(
+          clusterTypeGroups[key],
+          'representativeId'
+        );
+      });
+      return allClusterTypesGroups;
+    }
+    return null;
+  }, [data]);
+
+  if (loading) {
+    return <Loader />;
+  }
+  if (error) {
+    return <Message level="failure">{error.message}</Message>;
+  }
+  if (!clusterData) {
+    return null;
+  }
+
+  const nameAndId = getEntrySectionNameAndId(EntrySection.SimilarProteins);
+
+  // TODO get extra columns about the cluster members
+
+  return (
+    <div id={EntrySection.SimilarProteins}>
+      <Card title={nameAndId.name}>
+        <Tabs>
+          {Object.keys(clusterData).map((clusterType) => (
```

Comment:
This needs to be sorted per identity somehow. Tabs in the current website are "100% Identity", "90% Identity", "50% Identity"

Reply:
Part of this comment addressed now, still pending ordering